pyomo/contrib/appsi/solvers/highs.py

```diff
@@ -273,7 +273,8 @@ def _solve(self, timer: HierarchicalTimer):
                 self._warm_start()
             timer.start('optimize')
             ostreams[-1].write("RUN!\n")
-            self._solver_model.HandleKeyboardInterrupt = True
+            if self.version()[0] >= 1 and self.version()[1] >= 8:
+                self._solver_model.HandleKeyboardInterrupt = True
```

Comment:
FWIW, there is a bug in this logic, as version 2.0 will not `HandleKeyboardInterrupt`.  An improved implementation would be:
```
if self.version()[:2] >= (1, 8):
```

Reply:
Good catch @jsiirola.

Reply:
creating a new PR now